Automation_Test_Framework/Asserter/Asserter.cs

```diff
@@ -0,0 +1,168 @@
+﻿using Core_Framework.DriverCore;
+using FluentAssertions;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using TheRookiesApp.DAO;
+
+namespace AssetManagementTestProject.Asserter;
+/// <summary>
+/// Assertions for strings/DAO/List DAO objects
+/// Also check order
+/// Alternatives if FluentAssertions fails
+/// </summary>
+public class Asserter : WebDriverBase
+{
+    public Asserter(IWebDriver driver) : base(driver)
```

Comment:
??? why asserter is inherited by WebDriverBase

Reply:
I define AssertEquals in WebDriverBase so Asserter inherit WebDriverBase